ui/js/directives/treeherder/clonejobs.js

```diff
@@ -900,7 +898,10 @@ treeherder.directive('thCloneJobs', [
         };
 
         var generateJobElements = function(resultsetAggregateId, resultset) {
-
+            ReactDOM.render(
+              React.createElement(window.Hello, {name: "Let's ReactJS!"}),
+              document.getElementsByClassName("job-list")[0]
```

Comment:
@maurodoglio, well it's working for me so I guess it would be there. I'll keep that in mind as I go along. Also, is it a good idea to use `document.getElementById` ?